GOVERNANCE.md

```diff
@@ -160,6 +160,30 @@ The roles used in this document are custom roles mapped according to the [GitHub
 
 If any of the above roles hasn't contributed in any phases (including, but not limited to: code changes, doc updates, issue discussions) in 3 months, the administrator needs to inform the member and remove one's roles and GitHub permissions.
 
+## Maintainers Areas
+
+The Kyverno code base covers many areas and project maintainers are not required to know everything about the project.
+For this reason, maintainers are specific to one (or more) area of the code base.
+Every area represent a specific aspect of the project.
+
+### Kyverno areas
+
+- API: The Kyverno APIs exposed to users
+- Helm chart: Packaging of the Kyverno project
+- Engine: The Kyverno policy engine
+- Reporting: The Kyverno reporting system
+- Webhooks configuration: Configuration of the webhooks based on policies present in the cluster
+- Cleanup: Cleanup policies engine
+- Image verification: Integration with Cosign and Notary
```

Comment:
Since we are adding "Cleanup" and "Image Verification" rules, may be good to also have "Validation Rules" and "Mutate and Generate Rules" as categories?